Traceback (most recent call last):
  File "./script", line 217, in <module>
    run_script()
  File "./script", line 196, in run_script
    contact="ome-users@lists.openmicroscopy.org.uk",
  File "/opt/omero/server/venv3/lib64/python3.6/site-packages/omero/scripts.py", line 408, in client
    handleParse(c)  # May throw
  File "/opt/omero/server/venv3/lib64/python3.6/site-packages/omero/scripts.py", line 425, in handleParse
    raise ParseExit(c.params)
omero.scripts.ParseExit
!! 09/29/21 13:41:56.618 error: communicator not destroyed during global destruction.